demo-ehr-app/src/utils/cds_response_constructor.ts

```diff
@@ -0,0 +1,42 @@
+import { CDS_SERVICE_SAMPLE_RESPONSE } from "../constants/data";
+
+export async function constructCDSResponse(hook: string, cdsRequest: any, showResponse: boolean = true) {
+
+  if (hook === "" || hook === undefined || !showResponse) {
+    return {
+      "cards": []
+    }
+  }
+
+  if (hook === "cds-services"){
+    return CDS_SERVICE_SAMPLE_RESPONSE;
+  }
+
+  // make an api call to localhost:8080/cds-services
+  const response = await fetch("http://localhost:8080/cds-services/" + hook, {
```

Comment:
Is this valid for Choreo deployments as well?

Reply:
https://github.com/wso2-enterprise/open-healthcare/issues/1675